src/actions/breakpoints/addBreakpoint.js

```diff
@@ -101,7 +101,7 @@ async function addBreakpointPromise(getState, client, sourceMaps, breakpoint) {
 
   const newBreakpoint = {
     id: makeLocationId(generatedLocation),
-    disabled: false,
+    disabled: breakpoint.disabled,
```

Comment:
`breakpoint` is the data with options, which is why I'm using its `disabled` property.  We can't simply default this to `false`

Reply:
newBreakpoint needs to be false or the checkbox breaks.  thats why i manually set it to false if the shiftkey is pressed in my code.

Reply:
Right; this is the current problem; it's as though this is resetting the value somehow